validator/controller.go

```diff
@@ -285,40 +306,28 @@ func (c *controller) GetValidatorsIndices() []spec.ValidatorIndex {
 }
 
 // handleValidatorAddedEvent handles registry contract event for validator added
-func (c *controller) handleValidatorAddedEvent(validatorAddedEvent abiparser.ValidatorAddedEvent) error {
-	pubKey := hex.EncodeToString(validatorAddedEvent.PublicKey[:])
+func (c *controller) handleValidatorAddedEvent(validatorAddedEvent abiparser.ValidatorAddedEvent) (*validatorstorage.Share, error) {
```

Comment:
Let's change the name of the function, e.g. `createValidatorShare`

Reply:
we'll stick with the current name for now.
this area will be refactored in eth1_v2_events scope